### PR TITLE
Feature/100/producetasks

### PR DESCRIPTION
**Pull request by ZumzarHS**
## Quality check-list
-[x ] I have performed a self-review
-[ ] Unit tests have been created for the code 
-[ ] I have used linting/static code check

### CODE REVIEW

**Comment by TimMo-dev** [comment]
Det er en fin implementation, men det kan være det skal rettes igen ifbm. typeorm

**Comment by JakobTopholt** [comment]
> Det er en fin implementation, men det kan være det skal rettes igen ifbm. typeorm

Så du tænker at den ikke skal merges før typeorm er implementeret?

File: programming-learning-tool/src/SolutionPage.vue [hunk 1]
```diff
@@ -208,7 +235,7 @@ function navigate(path: string) {
                 </a>
                 <div class="bg-white h-full overflow-y-auto">
                   <div class="mx-4 my-8">
-                    {{feedbackOutput}}
+                    {{ feedbackOutput }}
```
Comment (JakobTopholt) [inline programming-learning-tool/src/SolutionPage.vue:238]: Bro why the spaces 😭

**Review by JakobTopholt — APPROVED** [review]
Yeah this is pretty solid.

**Pull request merged**